fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/UserInformationResult.java

```diff
@@ -46,96 +34,53 @@
     public UserInformationResult fintech_can_read_anton_brueckner_accounts_and_transactions() {
         ExtractableResponse<Response> response = withDefaultHeaders()
                                                          .given()
-                                                             .header(X_XSRF_TOKEN, UUID.randomUUID().toString())
-                                                             .header(SESSION_COOKIE, UUID.randomUUID().toString())
+                                                         .header(X_XSRF_TOKEN, UUID.randomUUID().toString())
+                                                         .header(SESSION_COOKIE, UUID.randomUUID().toString())
                                                          .when()
-                                                             .get(ACCOUNT )
+                                                         .get(ACCOUNT)
                                                          .then()
-                                                             .statusCode(HttpStatus.OK.value())
-                                                             .extract();
+                                                         .statusCode(HttpStatus.OK.value())
+                                                         .extract();
         this.respContent = response.body().asString();
         return (UserInformationResult) self();
     }
 
-    public UserInformationResult fintech_can_read_user_accounts_and_transactions() {
+    @SneakyThrows
+    public UserInformationResult fintech_can_read_users_transactions() {
         ExtractableResponse<Response> response = withDefaultHeaders()
                                                          .given()
-                                                             .header(X_XSRF_TOKEN, UUID.randomUUID().toString())
-                                                             .header(SESSION_COOKIE, UUID.randomUUID().toString())
+                                                         .header(X_XSRF_TOKEN, UUID.randomUUID().toString())
+                                                         .header(SESSION_COOKIE, UUID.randomUUID().toString())
                                                          .when()
-                                                             .get(ACCOUNT )
+                                                         .get(PAYMENT)
```

Comment:
This request tries to read user's payment status, instead, it should read users' accounts and transactions according to function name `fintech_can_read_users_transactions`. So either the function should be renamed or its body.